apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -90,9 +90,13 @@ const ProfileView = () => {
 
   const { data: user, isLoading } = trpc.viewer.me.useQuery(undefined, {
     onSuccess: (userData) => {
-      fetch(userData.avatar).then((res) => {
-        if (res.url) setFetchedImgSrc(res.url);
-      });
+      if (!userData.organization) {
+        fetch(userData.avatar).then((res) => {
+          if (res.url) setFetchedImgSrc(res.url);
+        });
+      } else {
+        setFetchedImgSrc("ORG_PROFILE_IMAGE");
+      }
```

Comment:
don't need this for org

Reply:
fetchedImgSrc was used to check if user has any set any image or not. url ending with /avatar.svg is allback image. (earlier it was gravatar url )

Reply:
NIT: feels weird to set a state to an unused string value, isn't there any other way? 🤔